fsharp-backend/src/Prelude/Prelude.fs

```diff
@@ -1083,6 +1083,7 @@ module Json =
     let assertSerializable (t : System.Type) : unit =
       if not (isSerializable t) then
         rollbarExceptionInBackground
+          false
```

Comment:
I didn't mean this should be an info, error is fine for `allow`. Sorry for the confusion.